### PR TITLE
Handle required fault domain configuration

### PR DESCRIPTION
**Pull request by adamtheturtle**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -75,21 +75,23 @@ def test_ee_example():
     superuser_username = str(uuid.uuid4())
     superuser_password = str(uuid.uuid4())
 
+    # DC/OS Enterprise clusters require various configuration variables which
+    # are not allowed or required by DC/OS OSS clusters.
+    extra_config = {
+        'superuser_username': superuser_username,
+        'superuser_password_hash': sha512_crypt.hash(superuser_password),
+        'fault_domain_enabled': False,
```
Comment (mnaboka) [inline README.md]: this needs to be a lowercase string `'false'`

File: tests/test_enterprise.py [hunk 2]
```diff
@@ -32,6 +32,7 @@ def test_run_pytest(
         config = {
             'superuser_username': superuser_username,
             'superuser_password_hash': sha512_crypt.hash(superuser_password),
+            'fault_domain_enabled': False,
```
Comment (mnaboka) [inline tests/test_enterprise.py]: same here

File: tests/test_enterprise.py [hunk 3]
```diff
@@ -99,6 +100,7 @@ def test_copy_files_to_installer(
             'security': 'strict',
             'ca_certificate_path': str(installer_cert_path),
             'ca_certificate_key_path': str(installer_key_path),
+            'fault_domain_enabled': False,
```
Comment (mnaboka) [inline tests/test_enterprise.py]: and here

File: tests/test_enterprise.py [hunk 4]
```diff
@@ -154,6 +156,7 @@ def test_auth_with_cli(
         config = {
             'superuser_username': superuser_username,
             'superuser_password_hash': sha512_crypt.hash(superuser_password),
+            'fault_domain_enabled': False,
```
Comment (mnaboka) [inline tests/test_enterprise.py]: and here

**Comment by adamtheturtle** [comment]
@mnaboka I don't think that this does need the lowercase string false, the system handles that - as per the passing tests.

**Comment by mnaboka** [comment]
@adamtheturtle in the calc.py we defiantly use a string https://github.com/mesosphere/dcos-enterprise/blob/master/gen_extra/calc.py#L735
maybe it gets jsonified somewhere in your code? if the tests pass i am happy with it

**Comment by adamtheturtle** [comment]
@mnaboka yep, we dump to YAML:

```
        config_yaml = yaml.dump(data={**config, **extra_config})
        config_file_path = self._genconf_dir / 'config.yaml'
        config_file_path.write_text(data=config_yaml)
```

**Pull request merged**